symphony-bdk-core-invokers/symphony-bdk-core-invoker-api/src/main/java/com/symphony/bdk/core/api/invoker/ApiClient.java

```diff
@@ -0,0 +1,162 @@
+package com.symphony.bdk.core.api.invoker;
+
+import org.apiguardian.api.API;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.core.GenericType;
+
+/**
+ * Interface used to perform HTTP requests performed by the generated Swagger code.
+ */
+@API(status = API.Status.STABLE)
+public interface ApiClient {
```

Comment:
did we consider other http clients? (like feign for instance?)

Reply:
Yes, the one that we will implement soon is the [WebClient](https://www.baeldung.com/spring-5-webclient) for Spring integration

Reply:
That one because it is a reactive one has a terrible API (IMHO) but I hope we will hide the complexity of with our own API